plugin/CMakeLists.txt

```diff
@@ -33,9 +33,8 @@ set(HEADERS
     zeek-reporter.h)
 
 set(ZEEK_SCRIPTS
-    scripts/__load__.zeek
-    scripts/base/spicy/__load__.zeek
-    scripts/base/spicy/main.zeek
+    scripts/_Zeek/spicy/__load__.zeek
```

Comment:
This seems to work, though it isn't quite what I was hoping for. This is current structure:

```
# find scripts
scripts
scripts/misc
scripts/misc/record-spicy-batch.zeek
scripts/__load__.zeek
scripts/__preload__.zeek
scripts/_Zeek
scripts/_Zeek/spicy
scripts/_Zeek/spicy/main.zeek
scripts/_Zeek/spicy/__load__.zeek
scripts/_Zeek/__load__.zeek
```

I think this still has the problem of `scripts/__load__.zeek` being pulled in twice (because `script_dir` in `zkg.meta` hasn't changed); we just don't see that because it happens to be empty. We could just remove it, but IO think`script_dir` should really be pointing one level down. I was original hoping for this: set `script_dir` to `plugin/scripts/spicy`; then this layout:

```
scripts/spicy/__load__.zeek (contains @load ./base)
scripts/spicy/__preload__.zeek
scripts/spicy/base/main.zeek
scripts/spicy/base/__load__.zeek
scripts/spicy/misc/record-spicy-batch.zeek
```

 You might have tried this and it didn't work?